src/controllers/TalksController.php

```diff
@@ -124,11 +124,17 @@ public function postAction($request, $db)
                         $comment    = $comment_mapper->getCommentById($new_id);
                         $speakers   = $talk_mapper->getSpeakerEmailsByTalkId($talk_id);
                         $recipients = array();
+                        $commentAuthorEmail = $comments[0]['email'];
                         foreach ($speakers as $person) {
+                            if ($commentAuthorEmail == $person['email']) {
+                                continue;
+                            }
                             $recipients[] = $person['email'];
                         }
-                        $emailService = new TalkCommentEmailService($this->config, $recipients, $talk, $comment);
-                        $emailService->sendEmail();
+                        if (count($recipients) > 0) {
+                            $emailService = new TalkCommentEmailService($this->config, $recipients, $talk, $comment);
```

Comment:
2 mins, just checked further down stream, there is a foreach which will just not run if there are no emails